api/server/routecache/cacher.go

```diff
@@ -0,0 +1,101 @@
+package routecache
+
+import (
+	"context"
+	"errors"
+	"github.com/iron-io/functions/api/models"
+	"github.com/iron-io/runner/common"
+	"math"
+	"sync"
+)
+
+type Cacher interface {
+	PrimeCache(ctx context.Context)
```

Comment:
I'm not sure what you mean with this. I mean, with this interface the ower should do whatever they want.